docker/build_docker.sh

```diff
@@ -170,6 +170,23 @@ function docker_build {
         # del_requirements=yes
     fi
 
+    if [ -f "pyproject.toml" -a ! -f "requirements.txt" ]; then
+       if [ ! -f "poetry.lock" ]; then
+            echo "Error: pyproject.toml present without poetry.lock. Make sure to commit your poetry.lock file"
+            return 1
+        fi
+
+      curl -sSL https://install.python-poetry.org | python3 -  # download poetry
+      echo "starting to install dependencies in poetry..."
+      poetry --version
+      poetry self add poetry-plugin-export
```

Comment:
move this to dockerbuild also.
We should probably run `poetry env remove` here like we do for pipenv

Reply:
done, once https://github.com/demisto/dockerfiles/pull/10296 and https://github.com/demisto/dockerfiles/pull/9337 are merged ill remove this part.